Extend research to all text fields in Song object

Benstales opened the pull request:
Update simple query (the remaining field in the query_language.py) so that it allows to search according to the version, detail and detail_video fields from Song object.

Neraste commented:
Thanks for your contribution.

As you see, one test fails (`test_parse_old_worktype (library.test_query_language.QueryLanguageParserTestCase)`), so you have to adapt it to suit the modifications of this PR.

Moreover, you have to write tests for this feature to be accepted. You can take inspiration from the other tests (`test_query_language.py` and `tests_song.py`).

Benstales commented:
This time I have taken in consideration the test environment.
I reversed the changes from my first PR in the query_language.py since there were not relevant.
I modified the views.py so that it takes into account the detail fields in the remain field of the query language.
I modified song2 in the test base. Instead of making a new function in order to test this feature, I completed the test_get_song_list_with_query in tests_song.py since we're dealing with the remain in this issue.
